src/python/foglamp/admin_api/README.rst

```diff
@@ -1,39 +1,86 @@
 Introduction
 ------------
```

Comment:
These are good suggestions but I think this type of documentation will either end up on http://swaggerhub.com or will be in docstrings. Especially the "methods" section.